src/NetworkPortInstantiation.php

```diff
@@ -928,7 +928,7 @@ public static function showConnection($netport, $edit = false)
                         _x('button', 'Disconnect'),
                         ['id' => $relations_id],
                         'fa-unlink netport',
-                        'class="btn btn-sm btn-ghost-danger"'
```

Comment:
prefer an outline version to avoid attract attention on a secondary button.
i agree ghost was confusing in the current case (confusion mostly on color state with possible action)

Reply:
done =)